Model/CoEntitlementProvisionerTarget.php

```diff
@@ -335,23 +338,28 @@ public function provision($coProvisioningTargetData, $op, $provisioningData)
         
         // Check if its an action we want to provision
         if(!empty($_REQUEST['_method']) && $_REQUEST['_method'] == 'PUT' && !empty($_REQUEST['data']['CoPersonRole'] && $_REQUEST['data']['CoPersonRole']['status'] == 'S')) { //SUSPEND
```

Comment:
Move all these if statements to a separate function. Something like requestIdentify()